Cannot find module solid-plugin when building AUR package

### Description

Since this morning, version 1.0.51, I'm getting this when building the AUR package (https://aur.archlinux.org/packages/opencode):
```
error: Cannot find module '../node_modules/@opentui/solid/scripts/solid-plugin' from '/tmp/makepkg.builddir/opencode/src/opencode-1.0.51/packages/opencode/script/build.ts'

Bun v1.3.2 (Linux x64)
==> ERROR: A failure occurred in build().
    Aborting...
```

### OpenCode version

_No response_

### Steps to reproduce

_No response_

### Screenshot and/or share link

_No response_

### Operating System

_No response_

### Terminal

_No response_

--- a/.github/workflows/snapshot.yml
+++ b/.github/workflows/snapshot.yml
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - dev
-      - windows
+      - fix-build
       - v0
 
 concurrency: ${{ github.workflow }}-${{ github.ref }}

--- a/packages/opencode/script/build.ts
+++ b/packages/opencode/script/build.ts
@@ -1,6 +1,5 @@
 #!/usr/bin/env bun
 
-import solidPlugin from "../node_modules/@opentui/solid/scripts/solid-plugin"
 import path from "path"
 import fs from "fs"
 import { $ } from "bun"
@@ -9,6 +8,9 @@ import { fileURLToPath } from "url"
 const __filename = fileURLToPath(import.meta.url)
 const __dirname = path.dirname(__filename)
 const dir = path.resolve(__dirname, "..")
+
+const solidPluginPath = path.resolve(dir, "node_modules/@opentui/solid/scripts/solid-plugin.ts")
+const solidPlugin = (await import(solidPluginPath)).default
 
 process.chdir(dir)
 

